exercises/practice/raindrops/.meta/create_data

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
Should we use '.sh' extension for bash scripts, so it is easier to see what they are?

Reply:
If we're adding extensions, this would be .bash but in general, Unit executables don't tend to have extensions, eg `ls.elf`

Reply:
I don't mind: i just like the track to be consistent.